understanding/focus-not-obscured-minimum.html

```diff
@@ -380,90 +275,62 @@ <h4>User openable, persistent disclosures</h4>
                </section>
                <section id="dialogs">
                   
-                  <h3>Modal dialogs</h3>
+                  <h3>モーダルダイアログ</h3>
                   
                   
-                  <p>A properly constructed modal dialog will always pass this SC. Even if it appears directly
-                     on top of an item with focus, the dialog takes focus on appearance, and thus the item
-                     receiving focus -- the dialog or one of its components -- is visible. A properly constructed
-                     modal maintains that focus and prevents interaction outside the modal until it is
-                     dismissed.
+                  <p>適切に構成されたモーダルダイアログは、常にこの達成基準に合格する。たとえフォーカスを持つ項目の上に直接表示されたとしても、ダイアログは外観上フォーカスを受け取り、したがってフォーカスを受け取る項目 (ダイアログ又はそのコンポーネントの一つ) が表示される。適切に構成されたモーダルは、そのフォーカスを維持し、モーダルが閉じられるまでモーダル外のインタラクションを防ぐ。
```

Comment:
ここの "on appearance" は「表示された時に」というくらいのニュアンスのように思います ("on click" = クリック時、と同じニュアンス)。

"the item receiving focus" は「フォーカスを受け取る項目」となっていますが「フォーカスを受け取っている項目」としたほうが良いかもしれません。

```suggestion
                  <p>適切に構成されたモーダルダイアログは、常にこの達成基準に合格する。たとえフォーカスを持つ項目の上に直接表示されたとしても、表示された際にこのダイアログがフォーカスを受け取り、したがってフォーカスを受け取っている項目 (このダイアログ、又はその中のコンポーネントの一つ) は表示される。適切に構成されたモーダルは、そのフォーカスを維持し、モーダルが閉じられるまでモーダル外のインタラクションを防ぐ。
```